test/Libraries/Microsoft.Extensions.AI.Evaluation.NLP.Tests/RationalNumberTests.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Extensions.AI.Evaluation.NLP.Common;
+using Xunit;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.Tests;
+
+public class RationalNumberTests
+{
+    [Fact]
+    public void Constructor_StoresNumeratorAndDenominator()
+    {
+        var r = new RationalNumber(3, 4);
+        Assert.Equal(3, r.Numerator);
+        Assert.Equal(4, r.Denominator);
+    }
+
+    [Fact]
+    public void Constructor_ThrowsOnZeroDenominator()
+    {
+        Assert.Throws<DivideByZeroException>(() => new RationalNumber(1, 0));
+    }
+
+    [Theory]
+    [InlineData(1, 2, 0.5)]
+    [InlineData(-3, 4, -0.75)]
+    [InlineData(0, 5, 0.0)]
+    public void ToDouble_ReturnsExpected(int num, int denom, double expected)
+    {
+        var r = new RationalNumber(num, denom);
+        Assert.Equal(expected, r.ToDouble(), 6);
+    }
+
+    [Fact]
+    public void ToString_FormatsCorrectly()
```

Comment:
Same comment - consider deleting this test

```suggestion
    public void ToDebugString_FormatsCorrectly()
```